### ISSUE TITLE
Why still blowfish?

### ISSUE THREAD

**Issue by 0xf10e** [seanpont/passman#1]
Bruce Schneier was quoted in 2007 [ "At this point, though, I'm amazed [Blowfish is] still being used. If people ask, I recommend Twofish instead."](https://en.wikipedia.org/wiki/Blowfish_%28cipher%29#cite_ref-schneier-interview-dec-2007_5-0).
So why not Twofish ([not considered broken by Bruce in 2005](https://en.wikipedia.org/wiki/Twofish#cite_note-4)) or Threefish ([might be considered showing little cracks](https://en.wikipedia.org/wiki/Threefish#Security) but those are probably not bigger than [the cracks in AES](https://en.wikipedia.org/wiki/Advanced_Encryption_Standard#Security)) if you want to stay with Bruce?

Going further down this lane one has, of course, suggest something like using [scrypt](https://en.wikipedia.org/wiki/Scrypt) to key the symmetric cipher.
- [Twofish in Go](https://godoc.org/golang.org/x/crypto/twofish)
- [Threefish in Go](https://github.com/wernerd/Skein3Fish) (as linked to on [Schneier on Security](https://www.schneier.com/threefish.html))
- [scrypt in Go](https://godoc.org/golang.org/x/crypto/scrypt)

**Comment by seanpont** [comment]
You're right, passman is due for an update. I will make the encryption protocol configurable and the default will be scrypt.

**Comment by 0xf10e** [comment]
Sweet, looking fwd to this update  :)
